src/main/java/no/sikt/nva/channelregister/ChannelRegister.java

```diff
@@ -116,11 +135,12 @@ public Optional<ErrorDetails> getChannelRegisterErrors(DublinCore dublinCore,
     }
 
     public String lookUpInChannelRegisterForPublisher(Record record, String customer) {
-        var publisher = formatValue(record.getPublication().getPublicationContext().getBragePublisher());
-        if (extractedIdentifierFromPublishersIsPresent(publisher, customer)) {
-            return lookUpInPublisher(formatValue(publisher), customer);
-        }
-        return null;
```

Comment:
Refactoring this method, do not really know why we implemented it like this (it was me). First we looked up in channel register -> if value was present - we looked up for this value again and returned it, otherwise null.